src/components/base/JAssetSymbol/JAssetSymbol.js

```diff
@@ -14,11 +15,32 @@ type Props = {
 const getSymbolClassName = symbol =>
   SYMBOLS_AVAILABLE_CLASS_INDEX[symbol.toLowerCase()] || '-symbol-not-listed'
 
-const JAssetSymbol = ({
-  color,
-  symbol,
-}: Props) => (
-  <div className={`j-asset-symbol ${getSymbolClassName(symbol)} -${color}`} data-symbol={symbol} />
-)
+const files = require.context('../../../public/assets/tokens/blue', true, /.*\.svg$/)
+const icons = files.keys().map(x => files(x).default).reduce((result, {
+  id, url,
+}) => ({
+  ...result,
+  [id]: url,
+}), {})
+
+class JAssetSymbol extends PureComponent<Props> {
```

Comment:
From my point of view, we need to using `function declaration` for stateless and methodless components